web/tailwind.config.ts

```diff
@@ -33,29 +34,20 @@ module.exports = {
         }
       },
       margin: { // for the checkmarks used for regex validation in Filters/Advanced
-        "2.5": "0.625rem", // 10px, between mb-2 (8px) and mb-3 (12px)
+        "2.5": "0.625rem" // 10px, between mb-2 (8px) and mb-3 (12px)
       },
       textShadow: {
         DEFAULT: "0 2px 4px var(--tw-shadow-color)"
       },
       boxShadow: {
         table: "rgba(0, 0, 0, 0.1) 0px 4px 16px 0px"
       }
-    },
+    }
   },
   variants: {
-    extend: {},
+    extend: {}
   },
   plugins: [
-    require("@tailwindcss/forms"),
-    plugin(function ({ matchUtilities, theme }) {
-      // Pipe --tw-shadow-color (i.e. shadow-cyan-500/50) to our new text-shadow
-      // Credits: https://www.hyperui.dev/blog/text-shadow-with-tailwindcss
-      // Use it like: text-shadow shadow-cyan-500/50
-      matchUtilities(
-        { "text-shadow": (value) => ({ textShadow: value }) },
-        { values: theme("textShadow") }
-      );
-    }),
-  ],
-}
```

Comment:
Good question, I am not sure if we need it, but that being said I did search the files and found only 1 location that uses it and there is a note there by @stacksmash76 says:

```// NOTE(stacksmash76): added text-shadow only for the dark theme - light theme is fine contrast-wise when it comes to headings
// ideally, this would need a redesign``` 

It's up to you if you want to keep that I can add it back just let me know.

Reply:
I took a look at this and it's used for every `CollapsibleSection` we have.
It adds a shadow to the heading of the section which is barely noticeable to me when not hovering over it.
When hovering over it adds a nice effect but doesn't add to readability in my opinion.
I would propose getting rid of it but if anyone has doubts about it, just react with a thumbs down please.